internal/exec/stages/files/filesystemEntries.go

```diff
@@ -265,16 +279,44 @@ func (s stage) createEntries(fs types.Filesystem, files []filesystemEntry) error
 	}
 
 	u := util.Util{
-		DestDir: mnt,
 		Fetcher: s.Util.Fetcher,
 		Logger:  s.Logger,
 	}
 
 	for _, e := range files {
+		path := e.getPath()
+		if fs.Name == "root" && strings.HasPrefix(path, "/run") {
```

Comment:
When does this occur? Is the only thing we want to carry over the one systemd unit and the file with the list of things to relabel (which is written by special one-off code)?

Reply:
This is for the general case of supporting files in `run`, e.g.:

```
"files": [{
      "filesystem": "root",
      "path": "/run/foo/bar",
      "contents": { "source": "data:,example%20file%0A" },
      "mode": 600
    }]
```

Reply:
I don't think we should support that at all. Ignition runs on first boot only and `/run` is not persistent. It's also not on the "root" filesystem so it's misleading.

Reply:
I don't feel strongly either way, though isn't more likely that people would be surprised that Ignition seems to ignore `/run` entries on `root` than being surprised that `/run` entries don't show up at every boot? (Which is understood to be transient state).

Of course, it doesn't ignore it, but it just has no visible effect for the `root` filesystem since the mount shadows the directory.

Reply:
`/run` is not the same filesystem as `/`. Doing a special case here is more unexpected here imo and breaks Ignition's defined semantics. Ignition should be correct over intuitive (ideally both, but if we need to make a choice, we should _always_ choose correct).